docs/md/modules-sg.md

```diff
@@ -0,0 +1,332 @@
+# GDCWholeGenomeSomaticSingleSample
+
+## Inputs
+
+In addition to the standard workload request inputs:
+
+- `executor` : URL of the Cromwell service
+- `output`   : GCS URL prefix for output files
+- `pipeline` : literally `"GDCWholeGenomeSomaticSingleSample"`
+- `project`  : some tracking label you can choose
+
+a `GDCWholeGenomeSomaticSingleSample` workload
+requires the following inputs
+for each workflow.
+
+- `base_file_name`
+- `contamination_vcf_index`
+- `contamination_vcf`
+- `cram_ref_fasta_index`
+- `cram_ref_fasta`
+- `dbsnp_vcf_index`
+- `dbsnp_vcf`
+- `input_cram`
+
+Here is what those are.
+
+#### `base_file_name`
+
+The leaf name
+of a sample input or output path
+without the `.` suffix.
+The `base_file_name`
+is usually the same
+as the _sample name_
+and differs in every workflow.
```

Comment:
Why are the columns so short?

Reply:
Because that's the way everyone should write markup documentation.